formfactory/views.py

```diff
@@ -53,17 +73,26 @@ def get_prefix(self):
         return self.kwargs.get("slug", self.form_slug)
 
     def get_success_url(self):
+
+        # If the intial post was an AJAX call we append the ajax query to the
+        # url. Primarily for when we return another formfactory view that
+        # already has some logic for AJAX.
+        ajax = self.request.GET.get("ajax")
+        url = "%s"
+        if ajax == "true":
+            url = "%s?ajax=true"
         redirect_url = self.form_object.redirect_to or self.request.GET.get(
             SETTINGS["redirect-url-param-name"]) or self.redirect_to
         if redirect_url:
-            return redirect_url
-        return self.request.path_info
+            return url % redirect_url
+        return url % self.request.path_info
 
     def get_context_data(self, **kwargs):
         context = super(FactoryFormView, self).get_context_data(**kwargs)
         context.update({
-            "form_object": self.form_object
+            "object": self.form_object
```

Comment:
Personal preference and expectation based on what I see regularly. Object will always be the singular database object you expect from a view and everything else is explicitly named or name-spaced. 

We are working with form views that returns a form created in the db. So my exception from this form view is the form object that has been saved and an actual django form instance.